### PR TITLE
Change warning to ignore unreadable partitions to be a digest

### PR DESCRIPTION
**Pull request by Moe82**
This PR is for issue [#15285](https://github.com/prestodb/presto/issues/15285)

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java [hunk 1]
```diff
@@ -474,8 +475,14 @@ public CounterStat getHighMemorySplitSource()
                 }
 
                 results.add(new HivePartitionMetadata(hivePartition, Optional.of(partition), partitionSchemaDifference.build(), encryptionInformation));
+                numOfPartitions++;
```
Comment (highker) [inline presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java:478]: Drop this variable; just use `partitionBatch.size()` as an approximation.

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java [hunk 2]
```diff
@@ -379,6 +380,8 @@ public CounterStat getHighMemorySplitSource()
                     partitions);
 
             ImmutableList.Builder<HivePartitionMetadata> results = ImmutableList.builder();
+            int numOfPartitions = 0;
+            ArrayList<String> partitionsNotReadable = new ArrayList<String>();
```
Comment (highker) [inline presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java:384]: use ImmutableList.Builder

**Review by highker — COMMENTED** [review]
There are compilation and style check failures. Could you help to address them?

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java (hunk 2 above)
Comment (Moe82) [reply presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java:384]: @highker Thanks for the feedback. Are you saying I should change the partitionsNotReadable's type to Immutable.Builder<String>? partitionsNotReadable just holds the names of the partitions that are not readable. Considering that partName is of type String, I figured that ArrayList<String> would be appropriate.

**Comment by Moe82** [comment]
> There are compilation and style check failures. Could you help to address them?

@highker This is a first pull request, so a lot of this is very new to me. Can I find the compilation error by looking at the logs in "continuous-integration/travis-ci/pr"?

**Comment by Moe82** [comment]
> > There are compilation and style check failures. Could you help to address them?
> 
> @highker This is my first pull request, so a lot of this is very new to me. Can I find the compilation error by looking at the logs in "continuous-integration/travis-ci/pr"? How about style check failures?

I looked at the logs and I see the compilation and style errors now.

**Pull request closed without merging**